pkg/wallet/wallet.go

```diff
@@ -542,18 +631,21 @@ func (wallet *Wallet) GetNonce() string {
 }
 
 func addressFromPublicKey(pubKeyBytes []byte) string {
-	addr := blake3.Sum256(pubKeyBytes)
-	return UserAddressPrefix + base58.CheckEncode(addr[:], Base58Version)
+	versionedPubKey := append([]byte{PubKeyVersion}, pubKeyBytes...)
```

Comment:
You should use the helper instead of doing it manually here.

Reply:
done in https://github.com/massalabs/station-massa-wallet/pull/530/commits/0a5c7c0381859807eedbe1b36ccda300f64171ed